dids/src/test/kotlin/web5/sdk/dids/methods/key/DidKeyTest.kt

```diff
@@ -24,52 +21,25 @@ class DidKeyTest {
     @Test
     fun `it works`() {
```

Comment:
nit: slightly more descriptive test name please
```suggestion
    fun `creates and resolves did:key`() {
```

Reply:
done!